src/sqlancer/stonedb/oracle/StoneDBQueryPartitioningAggregateTester.java

```diff
@@ -0,0 +1,168 @@
+package sqlancer.stonedb.oracle;
+
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import sqlancer.ComparatorHelper;
+import sqlancer.IgnoreMeException;
+import sqlancer.Randomly;
+import sqlancer.common.ast.newast.NewAliasNode;
+import sqlancer.common.ast.newast.NewFunctionNode;
+import sqlancer.common.ast.newast.NewUnaryPostfixOperatorNode;
+import sqlancer.common.ast.newast.NewUnaryPrefixOperatorNode;
+import sqlancer.common.ast.newast.Node;
+import sqlancer.common.query.SQLQueryAdapter;
+import sqlancer.common.query.SQLancerResultSet;
+import sqlancer.stonedb.StoneDBProvider.StoneDBGlobalState;
+import sqlancer.stonedb.StoneDBSchema.StoneDBCompositeDataType;
+import sqlancer.stonedb.StoneDBSchema.StoneDBDataType;
+import sqlancer.stonedb.StoneDBToStringVisitor;
+import sqlancer.stonedb.ast.StoneDBExpression;
+import sqlancer.stonedb.ast.StoneDBSelect;
+import sqlancer.stonedb.gen.StoneDBExpressionGenerator.StoneDBAggregateFunction;
+import sqlancer.stonedb.gen.StoneDBExpressionGenerator.StoneDBCastOperation;
+import sqlancer.stonedb.gen.StoneDBExpressionGenerator.StoneDBUnaryPostfixOperator;
+import sqlancer.stonedb.gen.StoneDBExpressionGenerator.StoneDBUnaryPrefixOperator;
+
+public class StoneDBQueryPartitioningAggregateTester extends StoneDBQueryPartitioningBase {
```

Comment:
StoneDBAggregateOracle. Similar to other oracles.

Reply:
I name it after the DuckDB implementation. Should change the class name to `StoneDBAggregateOracle` or not?

Reply:
change